src/ResourceManager/DataMigration/Commands.DataMigration/help/New-AzureRmDataMigrationService.md

```diff
@@ -13,12 +13,12 @@ Creates a new instance of the Azure Database Migration Service.
 ## SYNTAX
 
 ```
-New-AzureRmDataMigrationService -ResourceGroupName <String> -ServiceName <String> -Location <String>
- -Sku <String> -VirtualSubnetId <String> [-DefaultProfile <IAzureContextContainer>]
+New-AzureRmDataMigrationService -ResourceGroupName <String> -Name <String> -Location <String> -Sku <String>
```

Comment:
You'll need to add an alias to "ServiceName" to prevent this from being a breaking change.

Reply:
Fixed